l10n_ar_withholding_ux/models/account_payment.py

```diff
@@ -73,7 +73,6 @@ def action_confirm(self):
                     'Por favor, compute las retenciones para que el importe a pagar se actualice y luego confirme el pago.' % (
                         previous_to_pay, rec.to_pay_amount
                     ))
-        self.filtered('company_id.automatic_withholdings').compute_withholdings()
```

Comment:
lets wait for maru feedback
* if we need to let the force update then change the line to `self.compute_withholdings()`